modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
+
+. Expand the next **Custom manifest** section and repeat the same steps for the following manifests:
+    - CSI driver manifest: `custom_ manifest/manifests/oci-csi.yml`
+    - CCM machine configuration: `custom_ manifest/openshift/machineconfig-ccm.yml`
+    - CSI driver machine configuration: `custom_ manifest/openshift/machineconfig-csi.yml`
+
+. Complete the **Review and create** step to create your {product-title} cluster on {oci}. Click the **Install cluster** button to finalize the cluster installation. 
+
+. Open the custom manifest template, `oci-ccm.yml`, and replace the cloud configuration section of the template file with the configuration you previously copied from the **Logs** section on {oci}. Perform the same steps to update the configuration of the CSI driver in the `oci-csi.yml` file.
++
+The following example replaced the `region:` to `rateLimitBucketWrite` parameters of the `oci-csi.yml` file with the configuration from the `oci_ccm_config` cloud configuration on {oci}. 
++
+[source,yaml]
+----
+#…
+config.yaml: |
+   auth: 
+     region: <region_name>
+     useInstancePrincipals: true
+     compartment: <compartment_ocid>
+   vcn: <virtual_cloud_network_ocid>
+   loadBalancer:
+      subnet1: <subnet_ocid>
+      securityListManagementMode: Frontend
+   rateLimiter:
+     rateLimitQPSRead: 20.0
+     rateLimitBucketRead: 5
+     rateLimitQPSWrite: 20.0
+     rateLimitBucketWrite: 5
+#…
+----
+
+. Go to **Services > Instances** on {oci}. Select your compartment, and then click the **Create instance** button. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Compute/Tasks/launchinginstance.htm#top[Creating an instance] in the Oracle documentation. 
+
+. Add users to the instance. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/Compute/Tasks/addingusers.htm#Adding_Users_on_an_Instance[Adding users to an instance] in the Oracle documentation. 
+
+
+.Verification
+
+. From the Hybrid Cloud Console, go to **Clusters > Assisted Clusters** and select your cluster’s name. 
+
+. Check that the **Installation progress** bar is at **100%** and a message displays indicating `Installation completed successfully`.
+
+. To access the {product-title} web console, click on the provided Web Console URL.
+
+. Go to the **Nodes** menu page. 
+
+. Locate your node from the **Nodes** table. 
+
+. From the **Overview** tab, check that your node has a `Ready` status. 
+
+. Select the **YAML** tab.
+
+. Check the **Labels** parameter, and verify that the listed labels apply to your configuration. For example, the `topology.kubernetes.io/region=us-sanjose-1` label indicates in what {oci} region the node was deployed.
```

Comment:
Nit. **Labels** is actually a key, not a parameter, but that's not a big deal. Markup should be backticks, not bold, because the word is in a YAML file, not a UI.